README.md

```diff
@@ -1,28 +1,124 @@
-# FaaS Acceptance Test Suite (FATS) for riff [![Build Status](https://travis-ci.org/projectriff/fats.svg?branch=master)](https://travis-ci.org/projectriff/fats)
-
-Uses travis-ci to run acceptance tests
-
-Travis will:
-- install and configure kubectl, docker, gcloud and helm
-- create a GKE cluster
-- install riff into GKE via helm
-- deploy, invoke and delete functions for:
-    - command
-    - go
-    - java
-    - node
-    - python3
-- deletes GKE cluster
-
-Attempts are made to cleanup all Google Cloud resources that are
-created, however, if the script errors the resources may be stuck.
-
-GKE clusters start with the prefix `ci-fats-`.
-Docker image repositories start with `gcr.io/cf-spring-pfs-dev/fats-`.
-
-The `pfs-ci@cf-spring-pfs-eng.iam.gserviceaccount.com` service account
-is used to access Google Cloud. The authentication key-file is provided
-as a base64 encoded secure env var named `GCLOUD_CLIENT_SECRET`. To
-rotate the credentials, update the secure env var section of the
-travis-ci settings with the value generated from
-`cat [downloaded keyfile].json | base64 | pbcopy`.
+# FaaS Acceptance Test Suite (FATS) for riff
+
+FATS is a suite of scripts that support testing riff against various Kubernetes clusters.
+
+## Running FATS
+
+FATS is expected to be driven by the repository being tested.
+
+An example config is provided in the `.travis.yml` file for this repo that doubles as a test suite for FATS itself. The configuration defines environment variables and scripts to invoke.
+
+- `CLUSTER_NAME` a short, url safe, unique name (e.g. fats-123-4) used to distinguish resource between concurrent runs.
+- `NAMESPACE` the namespace to install resources into. May be hard coded for clusters that are provisioned on demand or dynamic for clusters that are reused between runs. (Note: if resources are shared jobs should not run concurrently)
+- `CLUSTER` the type of cluster to use. Some clusters will require additional environment variables
+- `REGISTRY` the type of registry to use. Some registries will require addition environment variables
+
+There are two scripts that are commonly defined for Travis based builds, other CI environments may be configured differently.
+
+- `script` - starts the cluster then run your tests. Run `./start.sh` to start the cluster and registry.
+- `after_script` - runs after the script passes or fails. Run `./cleanup.sh` to cleanup the cluster and registry.
+
+FATS will:
+
+- configure and start a kubernetes cluster defined by `$CLUSTER`
+- configure and start an image registry defined by `$REGISTRY`
```

Comment:
What does it mean to "start" a registry like say GCR?

Reply:
I think "start" makes sense only for local registries (i.e. minikube at the moment)
GCR and Dockerhub start scripts are no-op.